### PR TITLE
Added titles and text to the subplots to make it easier to new comers for plot_censure.py example

### PR DESCRIPTION
**Pull request by rovitotv**
## Description

Modified the example code to include titles and some text to the subplots.  Trying to make it clear for new comers issue #1967. Not sure if the text in yellow with the AffineTransform parameters makes the plot more or less clear.  Seeking feedback.  Thanks!

### CODE REVIEW

File: doc/examples/features_detection/plot_censure.py [hunk 1]
```diff
@@ -31,12 +31,18 @@
...
 
 ax[1].imshow(img_warp, cmap=plt.cm.gray)
 ax[1].axis('off')
 ax[1].scatter(detector.keypoints[:, 1], detector.keypoints[:, 0],
               2 ** detector.scales, facecolors='none', edgecolors='r')
+ax[1].set_title("Transformed Image")
+
+ax[1].text(372, 450, 'AffineTransform scale = 1.5', ha='center', va='center', color='yellow')
+ax[1].text(325, 475, 'AffineTransform rotation 28.6 degrees', ha='center', va='center', color='yellow')
+ax[1].text(290, 500, 'AffineTransform translation X = 150  Y = -200', ha='center', va='center', color='yellow')
```
Comment (soupault) [inline doc/examples/features_detection/plot_censure.py]: I think it'd be better to output this text via `Output` cells (i.e. just use `print` function).
See the last plot and the cell below it in http://scikit-image.org/docs/dev/auto_examples/features_detection/plot_local_binary_pattern.html .

**Comment by emmanuelle** [comment]
LGTM :+1:

**Comment by ahojnnes** [comment]
@emmanuelle I think, the titles are a good addition, but the `print` statements do not add value to the example.

**Comment by shashankredemption** [comment]
I think this can be merged once the print statements are removed.

**Comment by rovitotv** [comment]
I removed the print statements as suggested....

**Comment by jni** [comment]
Thanks @rovitotv!

**Comment by rovitotv** [comment]
@jni thanks for merging!  Thanks also to @soupault and @emmanuelle and @shashankredemption for feedback and reviews.  These three commits were done during 2016 SciPy sprints I am glad to see them merged.  

Happy New Year!

**Pull request merged**